asreview/review/simulate.py

```diff
@@ -46,7 +46,7 @@ class ReviewSimulate(BaseReview):
     n_prior_excluded: int
         Sample n prior excluded papers.
     prior_idx: int
-        Prior indices by id.
+        Prior indices by row number.
```

Comment:
starting at zero or one? maybe be specific

Reply:
The clarification is available in the CLI help. Internally, this is fine because Python counts from 0 by default.